engine/api/src/main/scala/pl/touk/nussknacker/engine/api/defaults/ParameterDefaultValueDeterminer.scala

```diff
@@ -0,0 +1,9 @@
+package pl.touk.nussknacker.engine.api.defaults
+
+import pl.touk.nussknacker.engine.api.definition.Parameter
+
+trait ParameterDefaultValueDeterminer {
```

Comment:
Why is only this trait moved to api module and the rest stayed in ui module?

Reply:
I thought about it but then I realize that we don't expose in api ability to totally change chain but only first step - custom default values. And it is generally good approach because why someone would like to skip determining based on configuration from file? By expose in api I mean using `ServiceLoader`

Reply:
I didn't notice `ServiceLoader`. Then that's ok, I think